far/help.cpp

```diff
@@ -1429,7 +1433,7 @@ bool Help::ProcessKey(const Manager::Key& Key)
 			// не поганим SelTopic, если и так в FoundContents
 			if (StackData->strHelpTopic != FoundContents)
 			{
-				if (GetSearchReplaceString({}, LastSearchDlgParams, L"HelpSearch"sv, {}, CP_DEFAULT, {}, &HelpSearchId) == SearchReplaceDlgResult::Cancel)
+				if (GetSearchReplaceString({}, SearchDlgParams(), L"HelpSearch"sv, {}, CP_DEFAULT, {}, &HelpSearchId) == SearchReplaceDlgResult::Cancel)
```

Comment:
Can Help be opened recursively? I hope it cannot but wanted to double check. Otherwise, besides static Params, we need an instance copy of Params.

Reply:
F1 F7 F11 F1 F7 F11 F1 F7 F11 F1 F7 F11...

Reply:
Ugh! Will model Help Search params after all other components'.

Reply:
Done!